sdk/servicebus/azure-servicebus/azure/servicebus/aio/management/_management_client_async.py

```diff
@@ -0,0 +1,853 @@
+        # type: (str, bool, Any) -> ElementTree
+
+        with _handle_response_error():
+            element = cast(
+                ElementTree,
+                await self._impl.entity.get(entity_name, enrich=enrich, api_version=constants.API_VERSION, **kwargs)
+            )
+        return element
+
+    async def _get_subscription_element(self, topic_name, subscription_name, enrich=False, **kwargs):
+        # type: (str, str, bool, Any) -> ElementTree
+
+        with _handle_response_error():
+            element = cast(
+                ElementTree,
+                await self._impl.subscription.get(
+                    topic_name, subscription_name, enrich=enrich, api_version=constants.API_VERSION, **kwargs)
+            )
+        return element
+
+    async def _get_rule_element(self, topic_name, subscription_name, rule_name, **kwargs):
+        # type: (str, str, str, Any) -> ElementTree
+
+        with _handle_response_error():
+            element = cast(
+                ElementTree,
+                await self._impl.rule.get(
+                    topic_name, subscription_name, rule_name, enrich=False, api_version=constants.API_VERSION, **kwargs)
+            )
+        return element
+
+    async def get_queue(self, queue_name: str, **kwargs) -> QueueDescription:
+        """Get a QueueDescription.
+
+        :param str queue_name: The name of the queue.
+        :rtype: ~azure.servicebus.management.QueueDescription
+        """
+        entry_ele = await self._get_entity_element(queue_name, **kwargs)
+        entry = QueueDescriptionEntry.deserialize(entry_ele)
+        if not entry.content:
+            raise ResourceNotFoundError("Queue '{}' does not exist".format(queue_name))
+        queue_description = QueueDescription._from_internal_entity(
+            entry.content.queue_description)
+        queue_description.name = queue_name
+        return queue_description
+
+    async def get_queue_runtime_info(self, queue_name: str, **kwargs) -> QueueRuntimeInfo:
+        """Get the runtime information of a queue.
+
+        :param str queue_name: The name of the queue.
+        :rtype: ~azure.servicebus.management.QueueRuntimeInfo
+        """
+        entry_ele = await self._get_entity_element(queue_name, **kwargs)
+        entry = QueueDescriptionEntry.deserialize(entry_ele)
+        if not entry.content:
+            raise ResourceNotFoundError("Queue {} does not exist".format(queue_name))
+        runtime_info = QueueRuntimeInfo._from_internal_entity(
+            entry.content.queue_description)
+        runtime_info.name = queue_name
+        return runtime_info
+
+    async def create_queue(self, queue: Union[str, QueueDescription], **kwargs) -> QueueDescription:
+        """Create a queue.
+
+        :param queue: The queue name or a `QueueDescription` instance. When it's a str, it will be the name
+         of the created queue. Other properties of the created queue will have default values decided by the
```

Comment:
"decided by the service"

Reply:
Changed to "as defined by the service". Another comment from Kieran and Adam @ https://github.com/Azure/azure-sdk-for-python/pull/12227#discussion_r447964401